app/views/layouts/application.html.erb

```diff
@@ -1,14 +1,15 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>ToyApp</title>
+  <title>ToyApp</title>
+  <title><%= yield(:title) %> | Ruby on Rails Tutorial Sample App</title>
     <%= csrf_meta_tags %>
-
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
```

Comment:
- Dùng " thay cho ' trong toàn bộ file này
- Sau stylesheet_link_tag cách 1 space thôi